source/jormungandr/jormungandr/interfaces/v1/Journeys.py

```diff
@@ -479,6 +481,11 @@ def _set_specific_params(mod):
         else:
             _set_specific_params(default_values)
 
+        # set parameters when is_journey_schedules is set to True
+        if args.get("is_journey_schedules"):
+            args["min_nb_journeys"] = 5
```

Comment:
I don't think this should override the `min_nb_journeys`

And I'm not sure we should add this parameter under the hood, I would prefer this to be explicit in the `same_journey_schedule`